pkg/idutil/imagewalker/imagewalker.go

```diff
@@ -64,14 +64,26 @@ func (w *ImageWalker) Walk(ctx context.Context, req string) (int, error) {
 		return -1, err
 	}
 
-	matchCount := len(images)
 	// to handle the `rmi -f` case where returned images are different but
 	// have the same short prefix.
 	uniqueImages := make(map[digest.Digest]bool)
 	for _, image := range images {
 		uniqueImages[image.Target.Digest] = true
 	}
 
+	// Allow to nerdctl rmi <short digest ids of another images> to remove images.
+	if len(uniqueImages) > 1 {
```

Comment:
Is it working as intended?

There could be a case where you have multiple images with different digests, which digest start with the same few characters.
In that case, you would have multiple `uniqueImages`, and this would be find, we should untag them all.

Reply:
Thanks for comments!!!

The following if statement is necesary.

```go
if len(uniqueImages) > 1 {
...
}
```

Without this if statement, running "nerdctl rmi <IMAGE ID>" fails.